internal/sip/sip.go

```diff
@@ -126,3 +127,20 @@ func (s SIP) IsAIP() bool {
 func (s SIP) IsSIP() bool {
 	return s.Type == enums.SIPTypeBornDigitalSIP || s.Type == enums.SIPTypeDigitizedSIP
 }
+
+func (s SIP) HasValidName() bool {
+	yyyymmdd := "(\\d{4})(\\d{2})(\\d{2})"
+	alphnum := "[a-zA-Z0-9]"
+
+	if s.Type == enums.SIPTypeBornDigitalSIP {
+		match, _ := regexp.MatchString(fmt.Sprintf("^SIP_%s_%s+[_]?%s+$", yyyymmdd, alphnum, alphnum), s.Name())
```

Comment:
I'm not clear from @sallain's description in #122 if the check should allow additional text after the matching pattern (e.g. "SIP_20250101_someoffice_refno_additional") or not. 🤔

Reply:
I'm going to assume not for now.